interfaces/builtin/unity8.go

```diff
@@ -20,11 +20,99 @@
 package builtin
 
 import (
+	"bytes"
 	"fmt"
 
 	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/interfaces/dbus"
 )
 
+var unity8ConnectedPlugAppArmor = []byte(`
```

Comment:
Please use 'const' instead of 'var' for anything security policy related. There are some older interfaces that don't do this, but this is the new style.